packages/smooth_app/lib/pages/user_management/login_page.dart

```diff
@@ -90,211 +90,251 @@ class _LoginPageState extends State<LoginPage> with TraceableClientMixin {
-            child: Container(
-              alignment: Alignment.topCenter,
-              width: double.infinity,
-              padding: EdgeInsets.symmetric(
-                horizontal: size.width * 0.15,
-                vertical: size.width * 0.05,
-              ),
-              child: AutofillGroup(
-                child: Center(
-                  child: Column(
-                    crossAxisAlignment: CrossAxisAlignment.center,
-                    mainAxisAlignment: MainAxisAlignment.center,
-                    mainAxisSize: MainAxisSize.min,
-                    children: <Widget>[
-                      SvgPicture.asset(
-                        'assets/preferences/login.svg',
-                        height: MediaQuery.of(context).size.height * .15,
-                      ),
-                      Text(
-                        appLocalizations.sign_in_text,
-                        textAlign: TextAlign.center,
-                        style: theme.textTheme.headline1?.copyWith(
-                          fontSize: 20.0,
-                          fontWeight: FontWeight.w700,
-                          color: theme.colorScheme.onSurface,
-                        ),
-                      ),
-
-                      const SizedBox(
-                        height: LARGE_SPACE * 3,
-                      ),
-
-                      if (_wrongCredentials) ...<Widget>[
-                        SmoothCard(
-                          padding: const EdgeInsets.all(10.0),
-                          color: Colors.red,
-                          child: Text(appLocalizations.incorrect_credentials),
-                        ),
-                        const Spacer(
-                          flex: 1,
-                        )
-                      ],
-                      //Login
-                      SmoothTextFormField(
-                        type: TextFieldTypes.PLAIN_TEXT,
-                        controller: userIdController,
-                        hintText: appLocalizations.username_or_email,
-                        textColor: _customGrey,
-                        backgroundColor: _textFieldBackgroundColor,
-                        prefixIcon: const Icon(Icons.person),
-                        enabled: !_runningQuery,
-                        // Moves focus to the next field
-                        textInputAction: TextInputAction.next,
-                        autofillHints: const <String>[
-                          AutofillHints.username,
-                          AutofillHints.email,
-                        ],
-                        validator: (String? value) {
-                          if (value == null || value.isEmpty) {
-                            return appLocalizations
-                                .login_page_username_or_email;
-                          }
-                          return null;
-                        },
-                      ),
+      body: SafeArea(
+        child: Stack(
```

Comment:
I have to say I am not a big fan of Stacks they seem error prone to me. I would recommend keeping the old way of using a transparent AppBar with 0 elevation and setting extendBodyBehindAppBar in the Scaffold to true.